components/layout/Footer.jsx

```diff
@@ -0,0 +1,79 @@
+import SmallText from "@/components/ui/typography/SmallText";
+import CustomLink from "@/components/ui/CustomLink";
+import useScale from "@/hooks/useScale";
+
+export default function Footer() {
+  const { scale } = useScale();
+  const smallTextStyle =
+    scale > 1
```

Comment:
Why are we using this scale hook

Reply:
We used the scale hook because we needed to calculate the scale with which to transform the text size as the screen width changed. We put the logic in a hook because it was being used in multiple files.

Reply:
css has a scale property 

You can use it like so 

`zoom: 1.1;
zoom: 0.7;
`

Reply:
did you face any issue by using the css zoom property without calculating it using javascript ?

Reply:
On the original website, the scale was dynamic and dependent on the window width, which required us to use JavaScript to obtain the dynamic scale. While the 'zoom' property was functional, it resulted in overflow issues, so we opted to scale only the font size instead.

Reply:
Ok great makes sense